app/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -3,13 +3,39 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from "react";
-import { RouteComponentProps } from "@reach/router";
+import { RouteComponentProps, useParams } from "@reach/router";
 import AppLayoutWithSidebarAndData from "../AppLayoutWithSidebarAndData";
+import { useAnalysis } from "../../hooks";
 
 const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
+  const { slug } = useParams();
+  const { loading, error, result } = useAnalysis(slug);
```

Comment:
I think we may want to check for some truthiness _above_ the return statement - this page is useful when the visualization data is available and in #3759 we'll be disabling the link to it if it's not available (I imagine we'll use this hook elsewhere for that check if the experiment status means the visualization data could be available).

I think we could at least check if it's loading and show the loading page if so. I'm less certain of an error state, it might make sense to go ahead and let the `useExperiment` hook fire a request up to the server to display the experiment data with an error message like you have here for analysis data. 🤔

@emtwo is the Results page going to still be useful if the visualization data errors? If not, we might as well save a request here and not query for experiment data if `error` is truthy here.

Reply:
Experiment data might still be useful in the results page even if there is an error. We can still show things like branch names, probe sets, firefox versions, etc.

Reply:
@jodyheavener I asked Ana about this and she'd like the error message to include something along the lines of, "Please contact data science in #cirrus about this."

Sounds like yeah, let's let the experiment data be queried even if the visualization data errors. I'll also take back my loading page comment, it looks like the visualization API request and GQL query happen in parallel and the GQL query isn't requeried when the `result` becomes available so; LGTM here.

Though, I still think we can probably show the loading page, otherwise `useExperiment` is hit twice, once for `loading` from the visualization hook and another when `result` is ready because of the rerender.

Reply:
@LZoog cool, I'll update the error message (we may want to tweak when actual visualization UI work begins since I meant for this to be placeholder until then :D)